Client/MirScenes/GameScene.cs

```diff
@@ -9197,7 +9201,17 @@ protected override void CreateTexture()
             if (MapObject.MouseObject != null && !(MapObject.MouseObject is ItemObject))
                 MapObject.MouseObject.DrawName();
 
-            int offSet = 0;
+            int offSet = 0; 
+            
+            for (int i = 0; i < Objects.Count; i++)
```

Comment:
Please add this under a client setting which is defaulted to OFF. Not everyone may want this feature.